odim-controller/helmcharts/odimra-config/templates/configmaps.yaml

```diff
@@ -75,7 +75,14 @@ data:
     		"SystemCollection": [
     			"Chassis",
     			"LogServices",
-    			"WorkloadPerformanceAdvisor"
+                        "Managers",
+    			"WorkloadPerformanceAdvisor"                
+    		],
+            "ManagerCollection": [
+    			"Systems",
```

Comment:
why skip systems and chassis under managers ?

Reply:
Chassis and Systems are already handled in a separate process, this list is added for ignoring the contents to be added in DB under manager